device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/ReconnectionTask.java

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
It might be a static function.
```java
public final ReconnectionNotifier() {
   private final static String THREAD_NAME = "azure-iot-sdk-ReconnectionTask";
   public static void notifyDisconnectAsync(final Throwable connectionLossCause, final IotHubListener listener, final String connectionId) {
      new Thread(
         new Runnable() {
               @Override
                      public void run() {
                              listener.onConnectionLost(connectionLossCause, connectionId);
                     }
            },
           THREAD_NAME 
      ).start();
}
```

Reply:
Looks good, I added it